### DISCUSSION TITLE
「开源之书·共读」#42: 《合作的进化》及其延伸读物 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#49] [Announcements]
# 「开源之书·共读」#42: 《合作的进化》及其延伸读物 - 「开源之道」

https://opensourceway.community/posts/book-of-open-source/42th-the-evolution-of-cooperation/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

建议, 随着阅读的深入, 
每个相关图书和其它图书的关联也在被触发出来, 

值得对每个 `#图书` 标签的 md,
在发布时, 自动在尾部追加一个另外持续更新的思维图谱,

类似: https://io.101.camp 这种,
将当前 开源之道 读过/推荐过的图书之间各种关系 都绘制出来,
并给出图片热点, 指向豆瓣图书页面...

形成线上微型图书馆...
